confidential/__init__.py

```diff
@@ -1,4 +1,4 @@
 from confidential.secrets_manager import SecretsManager
 
 __all__ = ["SecretsManager"]
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
We initially were using this to store the version, but the version is now stored by Poetry in `pyproject.toml`. So we move this, and you'll need to bump the version in `pyproject.toml`.

Reply:
Addressed in c64ea5d and 6d2b6a0